packages/file_selector/file_selector_platform_interface/lib/src/types/types.dart

```diff
@@ -1,3 +1,2 @@
-export 'x_file/x_file.dart';
-
+export 'package:cross_file/cross_file.dart';
```

Comment:
Should we reexport cross_file from here, or should we ask app users to depend on cross_file themselves? How would this behave if we have multiple plugins that use cross_file? WDYT @stuartmorgan?

Reply:
Forcing users to manually depend on another package that is required for using this one seems problematic. I don't know what happens if two different packages export the same third package though. Presumably it just means it's accessible via both namespaces?

Reply:
> Presumably it just means it's accessible via both namespaces?

Yeah, I think you're right, we can leave this here as a convenience.